tests/cli/test_config.py

```diff
@@ -9,28 +11,50 @@
 from nucypher.config.node import NodeConfiguration
 
 
-def test_initialize_temp_configuration_directory(temp_config_root):
+def test_initialize_configuration_directory():
     runner = CliRunner()
 
+    # Use the system temporary storage area
     result = runner.invoke(cli, ['configure', 'init', '--temp'], input='Y', catch_exceptions=False)
     assert '/tmp' in result.output, "Configuration not in system temporary directory"
     assert NodeConfiguration.TEMP_CONFIGURATION_DIR_PREFIX in result.output
     assert result.exit_code == 0
 
-    result = runner.invoke(cli, ['configure', 'destroy', '--temp'], input='Y', catch_exceptions=False)
-    assert NodeConfiguration.TEMP_CONFIGURATION_DIR_PREFIX in result.output
-    assert result.exit_code == 0
+    # Use an custom configuration directory
+    custom_filepath = '/tmp/nucypher-tmp-test-custom'
 
-    args = ['configure', 'init', '--config-root', temp_config_root]
-    result = runner.invoke(cli, args, input='Y', catch_exceptions=False)
-    assert '/tmp' in result.output, "Configuration not in system temporary directory"
-    assert temp_config_root in result.output
-    assert result.exit_code == 0
+    try:
+        os.mkdir(custom_filepath)  # TODO: Create a top-level installation directory, instead of writing only subdirs
 
-    args = ['configure', 'destroy', '--config-root', temp_config_root]
-    result = runner.invoke(cli, args, input='Y', catch_exceptions=False)
-    assert temp_config_root in result.output
-    assert result.exit_code == 0
+        args = ['configure', 'init', '--config-root', custom_filepath]
+        result = runner.invoke(cli, args, input='Y', catch_exceptions=False)
```

Comment:
Great, finally testing the CLI!

Reply:
Total side note: I think it's a good idea to have debuggable entry points that are built with this syntax.  It's been great for me to poke around and learn.